### ISSUE TITLE
Can't retrieve package.json for specified version

### ISSUE THREAD

**Issue by egasimus** [doowb/npm-api#7]
In a REPL, this works, eventually printing package information:
```
new require('npm-api')().repo('micromatch').package().then(console.log)
```
And this doesn't work:
```
new require('npm-api')().repo('micromatch').version('3.1.10').then(console.log)
```
Instead, it throws the following error:

```
(node:3159) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'latest' of undefined
    at Function.<anonymous> (/home/user/Code/node_modules/npm-api/lib/models/repo.js:92:25)
    at Generator.next (<anonymous>)
    at onFulfilled (/home/user/Code/node_modules/co/index.js:65:19)
    at process._tickCallback (internal/process/next_tick.js:68:7)
```

Am I missing something?

**Comment by doowb** [comment]
This is a bug... if you'd like to submit a PR, please do. I think that passing `'all'` to `.package()` [here](https://github.com/doowb/npm-api/blob/def50c2bda94a2673cdb84e776332a35ce02400c/lib/models/repo.js#L91) is all that's needed.

You can also use `.package` directly as a work-around:

```js
new require('npm-api')().repo('micromatch').package('3.1.10').then(console.log);
```

I actually don't remember why I implemented `.version` the way I did. I think it was an attempt at caching the entire package file from the registry instead of pulling it down for each version that was requested.

**Comment by doowb** [comment]
@egasimus sorry this took so long to publish. It's currently published as `0.4.13` to include #10.

I'm also refactoring this library due to NPM deprecating some of their public endpoints. I think the `.package` and `.version` methods used in this issue are still working, but some other methods in this library are broken until I can do the refactor.
